tests/basic_event_manager.rs

```diff
@@ -104,8 +115,13 @@ fn test_single_threaded() {
         assert_eq!(counters[i], 1 & (triggered_subscribers.contains(&i) as u64));
     }
 
+    let id = app.subscribers_id[0];
+
     // Once the app does not need events anymore, the cleanup needs to be called.
     // This is particularly important when the app continues the execution, but event monitoring
     // is not necessary.
     app.cleanup();
+
+    assert!(app.event_manager.subscriber_ref(id).is_none());
+    assert!(app.event_manager.subscriber_mut(id).is_err());
```

Comment:
Can this check the error more explicitly? E.g. `assert_eq!(app.event_manager.subscriber_mut(id),Err(..))`